src/sentry/web/frontend/oauth_token.py

```diff
@@ -72,16 +93,17 @@ def _get_access_tokens(self, request, client_id):
         if grant.is_expired():
             return self.error(request, "invalid_grant", "grant expired")
 
+        redirect_uri = request.POST.get("redirect_uri")
         if not redirect_uri:
             redirect_uri = application.get_default_redirect_uri()
         elif grant.redirect_uri != redirect_uri:
             return self.error(request, "invalid_grant", "invalid redirect_uri")
 
-        access_token = ApiToken.from_grant(grant)
-        id_token = self._get_open_id_token(grant, request)
-        return self._process_token_details(access_token, id_token)
+        access_token = ApiToken.from_grant(grant=grant)
+        id_token = self._get_open_id_token(request=request, grant=grant)
+        return self._process_token_details(token=access_token, id_token=id_token)
```

Comment:
hmm....

so now we have nested private methods...

Any way we can flatten these out?
no real reason for these to be private either is there? (this is kinda a semantic detail i guess)